test2.js

```
@@ -0,0 +1,20 @@
+var a = [];
+var b = [1];
+var c = [1, 2];
+var d = [1,
+    2];
+var e = [function foo() {
+    dosomething();
+}];
+
+var foo = { "bar": "This is a bar.", "baz": { "qux": "This is a qux" }, "difficult": "to read" };
+
+typeof foo === "strnig"
```

Biome

[error] Invalid `typeof` comparison value: "strnig" is not a valid type name

not a valid type name

(lint/suspicious/useValidTypeof)